osgar/test_logger.py

```diff
@@ -32,7 +34,7 @@ def delayed_copy(src, dst, skip_size):
             f2.flush()
 
 
-class LoggerTest(unittest.TestCase):
+class LoggerStreamingTest(unittest.TestCase):
```

Comment:
Why did you changed the name of the test? Maybe we should split into separate files (probably yes, as it is not short)

Reply:
To better reflect what it actually does, now that we have two different log readers.